<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">QSM - Qt SlideShow Manager</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNewSlideshow"/>
    <addaction name="separator"/>
    <addaction name="actionReloadAllSlideshows"/>
    <addaction name="actionSaveAllSlideshows"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionOptions"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionQsmHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAboutQt"/>
    <addaction name="actionAboutQsm"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewSlideshow"/>
   <addaction name="actionSaveAllSlideshows"/>
   <addaction name="separator"/>
   <addaction name="actionOptions"/>
   <addaction name="separator"/>
   <addaction name="actionQsmHelp"/>
   <addaction name="actionAboutQsm"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="folderBrowserDockWidget">
   <property name="windowTitle">
    <string>Folder Browser</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QTreeView" name="folderBrowserTreeView">
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
   </widget>
  </widget>
  <widget class="QDockWidget" name="imageListDockWidget">
   <property name="windowTitle">
    <string>Image List</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="imageListWidget">
    <layout class="QVBoxLayout" name="imageListWidgetLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="imageListOptionsGroupBox">
       <property name="title">
        <string>Image List Options</string>
       </property>
       <layout class="QHBoxLayout" name="imageListOptionsGroupBoxLayout">
        <property name="margin">
         <number>5</number>
        </property>
        <item>
         <widget class="QComboBox" name="imageListSortComboBox">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>Name</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Name Desc</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Date</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Date Desc</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="includeSubfoldersCheckBox">
          <property name="text">
           <string>Include Subfolders</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="imageListView">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="slideshowListDockWidget">
   <property name="windowTitle">
    <string>Slideshow List</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="SlideshowListView" name="slideshowListView">
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
   </widget>
  </widget>
  <widget class="QDockWidget" name="slideshowImageListDockWidget">
   <property name="windowTitle">
    <string>Slideshow Image List</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="slideshowImageListWidget">
    <layout class="QVBoxLayout" name="slideshowImageListWidgetLayout">
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="slideshowOptionsGroupBox">
       <property name="title">
        <string>Slideshow Options</string>
       </property>
       <layout class="QVBoxLayout" name="slideshowOptionsLayout">
        <item>
         <layout class="QHBoxLayout" name="slideshowOptionsLayout1">
          <item>
           <widget class="QComboBox" name="slideshowSortComboBox">
            <property name="currentIndex">
             <number>4</number>
            </property>
            <item>
             <property name="text">
              <string>Name</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Name Desc</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Date</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Date Desc</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Unsorted</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="slideshowStartPushButton">
            <property name="statusTip">
             <string>Starts the Slideshow</string>
            </property>
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="slideshowOptionsLayout2">
          <item>
           <widget class="QPushButton" name="beginPushButton">
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/begin.png</normaloff>:/icons/begin.png</iconset>
            </property>
            <property name="autoRepeat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="multipleUpPushButton">
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/multipleUp.png</normaloff>:/icons/multipleUp.png</iconset>
            </property>
            <property name="autoRepeat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="upPushButton">
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/up.png</normaloff>:/icons/up.png</iconset>
            </property>
            <property name="autoRepeat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="downPushButton">
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/down.png</normaloff>:/icons/down.png</iconset>
            </property>
            <property name="autoRepeat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="multipleDownPushButton">
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/multipleDown.png</normaloff>:/icons/multipleDown.png</iconset>
            </property>
            <property name="autoRepeat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="endPushButton">
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/end.png</normaloff>:/icons/end.png</iconset>
            </property>
            <property name="autoRepeat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="slideshowImageListView">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit the application</string>
   </property>
   <property name="statusTip">
    <string>Exit the application</string>
   </property>
  </action>
  <action name="actionStatusbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Statusbar</string>
   </property>
   <property name="statusTip">
    <string>Statusbar</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/spanner_48.png</normaloff>:/icons/spanner_48.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Options...</string>
   </property>
   <property name="statusTip">
    <string>Options</string>
   </property>
  </action>
  <action name="actionQsmHelp">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/question_48.png</normaloff>:/icons/question_48.png</iconset>
   </property>
   <property name="text">
    <string>QSM &amp;Help...</string>
   </property>
   <property name="statusTip">
    <string>QSM Help</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/info_48.png</normaloff>:/icons/info_48.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About Qt...</string>
   </property>
   <property name="toolTip">
    <string>Show the Qt library's About box</string>
   </property>
   <property name="statusTip">
    <string>Show the Qt library's About box</string>
   </property>
  </action>
  <action name="actionAboutQsm">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/info_48.png</normaloff>:/icons/info_48.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About QSM...</string>
   </property>
   <property name="toolTip">
    <string>Show the application's About box</string>
   </property>
   <property name="statusTip">
    <string>Show the application's About box</string>
   </property>
  </action>
  <action name="actionNewSlideshow">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/add_48.png</normaloff>:/icons/add_48.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New Slideshow</string>
   </property>
   <property name="toolTip">
    <string>Creates an empty Slideshow</string>
   </property>
   <property name="statusTip">
    <string>Creates an empty Slideshow</string>
   </property>
  </action>
  <action name="actionSaveAllSlideshows">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/floppy_disk_48.png</normaloff>:/icons/floppy_disk_48.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save All Slideshows</string>
   </property>
   <property name="iconText">
    <string>Save All Slideshows</string>
   </property>
   <property name="toolTip">
    <string>Save All Slideshows</string>
   </property>
   <property name="statusTip">
    <string>Save All Slideshows</string>
   </property>
  </action>
  <action name="actionReloadAllSlideshows">
   <property name="text">
    <string>Re&amp;load All Slideshows</string>
   </property>
   <property name="statusTip">
    <string>Reload All Slideshows</string>
   </property>
  </action>
  <action name="actionRemoveSlideshow">
   <property name="text">
    <string>&amp;Remove Slideshow</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="actionRemoveImage">
   <property name="text">
    <string>&amp;Remove Image</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="actionRenameSlideshow">
   <property name="text">
    <string>Re&amp;name Slideshow</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="actionReloadSlideshow">
   <property name="text">
    <string>Re&amp;load Slideshow</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="actionSaveSlideshow">
   <property name="text">
    <string>&amp;Save Slideshow</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="actionCopyImagesToSlideshow">
   <property name="text">
    <string>&amp;Copy Images to Slideshow...</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="actionRemoveImageFromFileSystem">
   <property name="text">
    <string>Remove Image from &amp;File System</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SlideshowListView</class>
   <extends>QListView</extends>
   <header>slideshowlistview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
